src/dndbeyond_character.pyj

```diff
@@ -66,12 +66,18 @@ def rollSavingThrow():
     rollAbilityOrSavingThrow("ct-ability-saving-throws-pane", "saving-throw")
 
 def rollInitiative():
+    nonlocal character
     initiative = $(".ct-initiative-box__value").text()
     advantage = $(".ct-initiative-box__advantage").length > 0
     if initiative == "":
         initiative = $(".ct-combat-mobile__extra--initiative .ct-combat-mobile__extra-value").text()
         advantage = $(".ct-combat-mobile__advantage").length > 0
     #console.log("Initiative " + ("with" if advantage else "without") + " advantage : " + initiative)
+
+    if character.getGlobalSetting("initiative-tiebreaker", False):
+       dexterity_score = [x[2] for x in character._abilities if x[1] == 'DEX'][0]

```

Comment:
This will crash if there is no page DEX ability, which could happen if user loads their sheet on a tab other than the abilities (not entirely sure, but i remember there were cases when it wouldn't find abilities, that's why it always checks elsewhere uf it finds it in the abilities array)